AGPs/AGP-1.md

```diff
@@ -66,29 +66,50 @@ Proposals made to the Finance track must affect the movement of assets held by t
   * AGPL
   * MIT
   
-_Finance track budget_
+*Finance track budget*
 
-The total amount funded by Finance track AGPs shall not exceed 5% of the market value of the Association / Aragon Network treasury or 250,000 DAI, whichever is greater, in a given quarterly Aragon Network vote cycle. Recurring transfers approved in previous vote cycles shall be deducted from this budget as described below. The market value of the treasury, and spending relative to that, shall be measured in DAI at the closing time of every ANV.
+The Finance track budget is calculated from the Aragon Treasury value. The Aragon Treasury is denominated in DAI. It is defined as the total market value of the Aragon Association multi-sig, net of all existing liabilities. It is snapshotted and calculated at the beginning of each Aragon Network Vote.
 
-For any Aragon Network vote ballot, approved Finance track proposals will be processed in the following order:
+- If the Treasury Value has grown by more than DAI 1M between this Aragon Network Vote and the previous one: the budget for this given ANV is 5% of the Aragon Treasury value.
+- If the Treasury Value has not grown by more than DAI 1M between this Aragon Network Vote and the previous one: the spending is arbitrarily capped to DAI 1M. This cap is there order to ensure that the treasury doesn't decrease too quickly before Aragon starts growing or appreciating significantly.
```

Comment:
Currently 5% of the budget is >1M, as I understand this proposal assuming the treasury doesn't grow by 1M subsequent budgets would be capped at 1M per quarter even if this represents less that 5% of the budget. 

Also when calculating the difference between treasury value are you capturing this after accounting for the spending in the previous budget? EG current treasury is 20M, 2M is spent, 1M comes in, has the treasury value increased by 1M or decreased by 1M?